app/assets/scripts/a11n/signin.js

```diff
@@ -23,15 +26,68 @@ import { FormikInputText } from '../components/common/forms/input-text';
 import { createProcessToast } from '../components/common/toasts';
 import { useUser } from '../context/user';
 
+const Backdrop = styled.div`
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: rgba(0, 0, 0, 0.5);
+  z-index: 1111;
```

Comment:
Can these be moved to a css/scss file? There are policies in place that restrict (albeit just warning as of now) use of inline css.

Reply:
@xhagrg We're using the styled components in this project (i.e. we don't have any css/scss files yet). This is the method we're using currently and is slightly different than the inline css (i.e. It is wrapped around the `styled` keyword which creates a component with those styles). There are some inline styles in this file, some of which is actually needed but I'll remove the ones that can be moved to styled components.